scripts/upgrade

```diff
@@ -11,48 +11,81 @@ source /usr/share/yunohost/helpers
 
 upgrade_type=$(ynh_check_app_version_changed)
 
+if ynh_compare_current_package_version --comparison le --version '14.10.8~ynh1'; then
+    ynh_die --mesage='Upgrade from this version is not supported. Please backup and restore manually the data after a clean new install'
+fi
```

Comment:
We may not need this as nobody as installed the app yet.

Reply:
Yes I know but it just to avoid issues with people which tried to test it and to be more explicit.